rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2587,6 +2587,56 @@ void voteFederationChange_addSameFederatorPublicKeyBySameAuthorizerTwice_returns
 
         }
 
+        @Test
+        void voteFederationChange_addMultiFederatorPublicKeyWithDifferentMstKey_returnsSuccessResponseCodeAndFedSizeZero() {
```

Comment:
nit: in a previous test you said `addFederatorMultiKey` instead of `addMultiFederatorPublicKey` (and sounds better imho)

Reply:
Updated.